tigerpath/majors_and_certificates/majors/CEE.json

```diff
@@ -41,14 +41,23 @@
               "min_needed": null,
               "explanation": null,
               "course_list": [
-                "CEE 205: Mechanics of Solids",
-                "CEE 262A: Structures and the Urban Environment",
+                "CEE 262B: Structures and the Urban Environment",
                 "CEE 312: Statics of Structures",
                 "CEE 361: Matrix Structural Analysis and Introduction to Finite-Element Methods",
                 "CEE 364: Materials in Civil Engineering",
                 "CEE 365: Soil Mechanics"
               ]
             },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CEE 205: Mechanics of Solids",
```

Comment:
https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering

This same change occurs multiple times